Sources/WoDoTri/WorkoutMapViewController.swift

```diff
@@ -1,5 +1,41 @@
 import UIKit
+import MapKit
 
 final class WorkoutMapViewController: UIViewController {
 
+    @IBOutlet weak private var mapView: MKMapView!
+    @IBOutlet weak private var centerOnMap: UIButton!
+
+    @IBAction func centerOnMapTapped(_ sender: UIButton) {
+        mapView.userTrackingMode = .followWithHeading
+        centerOnMap.isHidden = true
+    }
+    override func viewDidLoad() {
+
+        let mapDragRecognizer = UIPanGestureRecognizer(target: self, action: #selector(self.didDragMap(gestureRecognizer:)))
+        mapDragRecognizer.delegate = self
+
+        centerOnMap.isHidden = true
+        mapView.userTrackingMode = .followWithHeading
+        mapView.showsScale = true
+        mapView.showsCompass = true
+        mapView.isZoomEnabled = false
+        mapView.addGestureRecognizer(mapDragRecognizer)
+
+    }
+    @objc func didDragMap(gestureRecognizer: UIGestureRecognizer) {
+        if gestureRecognizer.state == UIGestureRecognizerState.began {
```

Comment:
Unnecessary explicit type, i.e. `UIGestureRecognizerState.began` can be inferred from `.began`.